src/manifest.ts

```diff
@@ -28,6 +28,7 @@ const manifest: ManifestType = {
         "*://*.moonscan.io/*",
         "*://*.gnosisscan.io/*",
         "*://*.bobascan.com/*",
+        "*://*.twitter.com/*",
```

Comment:
while at it id add all the other etherscans that we missed and x.com in case elmo has a divine revelation and decides to swap domains

Reply:
need x.com yeh

Reply:
added more etherscan EaaS domains and x.com, @0xStrobe I was unable to verify the new domains, can you give it a try?